### ISSUE TITLE
Fail to upload new sources

### ISSUE THREAD

**Issue by thomaspiller** [fxi/map-x-mgl#369]
#### Console
```
api.mapx.localhost:8880/upload/vector/:1 POST http://api.mapx.localhost:8880/upload/vector/ 500 (Internal Server Error)
```

#### Logs
```
api_1        | TypeError: Cannot read property 'search' of undefined
api_1        |     at fileToPostgres (/api/utils/uploadVector.js:226:24)
api_1        |     at convertOgrHandler (/api/utils/uploadVector.js:68:3)
api_1        |     at Layer.handle [as handle_request] (/api/node_modules/express/lib/router/layer.js:95:5)
api_1        |     at next (/api/node_modules/express/lib/router/route.js:137:13)
api_1        |     at validateToken.then.then.then.pData (/api/utils/authentification.js:85:9)
api_1        |     at <anonymous>
api_1        |     at process._tickCallback (internal/process/next_tick.js:188:7)
```
MapX v1.5.30 (local version)
Windows 10 Pro 64-bit
Chrome 70.0.3538.110 (Official Build) (64-bit)
No add-ons

**Comment by fxi** [comment]
Solved in lastest 1.5.30, it was an issue with the new common authentification system
Please test and close this if ok.

**Comment by thomaspiller** [comment]
Problem solved.
